visionos/ReactTestApp.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,565 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 54;
+	objects = {
+
+/* Begin PBXBuildFile section */
+		192DD201240FCAF5004E9CEB /* Manifest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 192DD200240FCAF5004E9CEB /* Manifest.swift */; };
```

Comment:
I took a lot (probably too much) of care to make sure the UUIDs here match the iOS pbxproj, just for consistency sake.